Common/Core/CollisionAssociation.h

```diff
@@ -0,0 +1,240 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file CollisionAssociation.h
+/// \brief Class for track to collision associators
+/// \author Jan Fiete Grosse-Oetringhaus <jan.fiete.grosse-oetringhaus@cern.ch>, CERN
+/// \author Fabrizio Grosa <fgrosa@cern.ch>, CERN
+/// \author Mattia Faggin <mfaggin@cern.ch>, University and INFN Padova
+/// \author Sarah Herrmann <sarah.herrmann@cern.ch>, IP2I Lyon
+/// \author Maurice Coquet <maurice.louis.coquet@cern.ch>, CEA-Saclay/Irfu
+
+#ifndef COMMON_CORE_COLLISIONASSOCIATION_H_
+#define COMMON_CORE_COLLISIONASSOCIATION_H_
+
+#include "CommonConstants/LHCConstants.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/ASoAHelpers.h"
+
+namespace o2::aod
+{
+namespace track_association
+{
```

Comment:
No major reasons, it allowed me to avoid to use the namespace of `track_association` inside the class, but I removed it if you prefer to have it outside.